CHANGELOG.md

```diff
@@ -65,10 +65,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - By default, only an Actor's replicated owner hierarchy will be used when determining which worker should have authority over an actor. Non-replicated Actors are now ignored.
 - Fixed a crash that would sometimes occur when connection to SpatialOS fails.
 - Fixed a crash that occurred when an actor subobject became invalid after applying initial component data.
-- Non-replicated Actors net roles are not touched during startup. 
+- Non-replicated Actors net roles are not touched during startup.
```

Comment:
```suggestion
- Non-replicated Actors net roles are not touched during startup. 
```

Reply:
I wonder if Visual Studio is making these changes?

Reply:
Maybe we could commit them and get them out of the way so it stops coming up :D

Reply:
Rider automatically linted these changes yes. As those spaces were not supposed to be here I figured I would let these changes here, so that IDE's won't edit that all the time

Reply:
yeah, I think that is fine :) 
we maybe need to add a linter for md files at some point